Handbook: Bizops rituals in people section (#5284)
fleetdm/fleet: handbook/people.md

      link: drive.google.com/[destinationFolderID]
   ```

> ## PeopleOps Rituals

Reviewer: ```suggestion
## PeopleOps
```

- Each Friday, if there are any upcoming workiversaries in the next 7 days, People Operations posts about them in #g-people and tags @mikermcneil to let them know.

> ## BizOps Rituals

Reviewer: ```suggestion
## Rituals
```

> The following rituals are engaged in by the  directly responsible individual (DRI) and at the frequency specified for the ritual.

Reviewer: ```suggestion
The following table lists the People group's rituals, frequency, and Directly Responsible Individual (DRI).
```

- Each Friday, if there are any upcoming workiversaries in the next 7 days, People Operations posts about them in #g-people and tags @mikermcneil to let them know.

## Rituals

The following table lists the People group's rituals, frequency, and Directly Responsible Individual (DRI).

| Ritual                       | Frequency                | Description                                         | DRI               |
|:-----------------------------|:-----------------------------|:----------------------------------------------------|-------------------|
| AP invoice monitoring | Daily | Look for new accounts payable invoices and ensure that Fleet's suppliers are paid. | Nathanael Holliday |
| Weekly department updates | Weekly | Updates from managers on what their departments accomplished for the week are logged in the cloud and disseminated on Slack. | Charlie Chance | 
| Hours update | Weekly | Screenshots of contractor hours as shown in Gusto are sent via Slack to each contractor's manager with no further action necessary if everything appears normal. | Charlie Chance |

> | 1:1 Mike:Sid Document Prep | Bi-weekly | Run through the document preparation GitHub issue for Mike's call with Sid. | Nathanael Holliday |

Reviewer: ```suggestion
| Prepare Mike and Sid's 1:1 Doc | Bi-weekly | Run through the document preparation GitHub issue for Mike's call with Sid. | Nathanael Holliday |
```

> | Brex reconciliation | Montly | Make sure all company issued credit card transactions have memos explaining what the purchase is for. | Nathanael Holliday |

Reviewer: ```suggestion
| OKR review | Every three weeks | Review the status of each OKR. | Mike Thomas |
| Brex reconciliation | Monthly | Make sure all company-issued credit card transactions include memos. | Nathanael Holliday |
```

| Monthly accounting | Monthly | Use the monthly accounting template in GitHub to go through the process of validating Fleet's books. | Nathanael Holliday |
| US contractor payroll | Monthly | Sync contractor hours to payments in Gusto and run payroll for the month. | Charlie Chance |

> | OKR planning | Quarterly and every three weeks | Review the status of each OKR. | |

Reviewer: ```suggestion
| OKR planning | Quarterly | Plan the next quarter's OKRs. | Mike Thomas |
```
